### PR TITLE
Python dev

### PR DESCRIPTION
**Pull request by mickael-guene**
Add an extension to allow write of extension from python.

### CODE REVIEW

**Comment by cedric-vincent** [comment]
Hello @mickael-guene,

Could you please add an example?  This should help newcomers to
understand your interface, also I will use this example as a test-case
to ensure I don't break your work.

C�dric.

File: tests/test-python01.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
+if [ -z `which mcookie` ] ||  [ -z `which cat` ] || [ -z `which grep` ] || [ -z `which rm` ] || [ -z `which find` ]; then
+    exit 125;
+fi
+
+if [ ! -e $PROOT ]; then
+    exit 125;
+fi
+
+if [ ! `$PROOT | grep '\-p' | grep string` ]; then
```
[the remark this answers is not in the document]
Comment (oxr463) [reply tests/test-python01.sh]: Not sure what is meant by "outside []", perhaps you could clarify?

```shell
if [ ! $("$PROOT --help" | grep -- -p | grep string) ]; then
```

File: tests/test-python01.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
+if [ -z `which mcookie` ] ||  [ -z `which cat` ] || [ -z `which grep` ] || [ -z `which rm` ] || [ -z `which find` ]; then
+    exit 125;
+fi
+
+if [ ! -e $PROOT ]; then
```
[the remark this answers is not in the document]
Comment (oxr463) [reply tests/test-python01.sh]:
```shell
if [ ! -e "$PROOT" ]; then
```

**Comment by oxr463** [comment]
- [ ] Needs rebase on master

**Pull request merged**